tests/integration/bond_test.py

```diff
@@ -277,4 +277,49 @@ def return_bond_and_slaves_mac():
 
     return bond99_bond_and_slaves_state[Interface.MAC], \
         eth1_bond_and_slaves_state[Interface.MAC], \
-        eth2_bond_and_slaves_state[Interface.MAC]
\ No newline at end of file
+        eth2_bond_and_slaves_state[Interface.MAC]
+
+
+@pytest.fixture
+def bond99():
+    with bond_interface(name='bond99', slaves=[]) as bond_state:
+        yield bond_state
+
+
+def test_bond_and_no_slaves_have_same_mac(bond99):
+    bond99_slaves_mac, eth1_slaves_mac, eth2_slaves_mac = \
+        return_bond_and_slaves_mac()
+
+    assert not bond99_slaves_mac == eth1_slaves_mac
```

Comment:
Use `!=` please (might be a duplicate comment, but I cannot see the other comment currently).